workers/data_refinery_workers/downloaders/test_array_express.py

```diff
@@ -90,15 +90,16 @@ def test_download(self,
         # Call the task we're testing:
         array_express.download_array_express.apply(args=(downloader_job.id,)).get()
 
-        target_file_path = ("/home/user/data_store/temp/A-AFFY-1/AFFY_TO_PCL/{}"
+        target_file_path = ("/home/user/data_store/temp/A-AFFY-1/AFFY_TO_PCL/downloader_job_{}"
                             "/E-GEOD-59071.raw.3.zip").format(str(downloader_job.id))
 
         # Verify that all expected functionality is run:
-        _verify_batch_grouping.assert_called_once()
+        # _verify_batch_grouping.assert_called_once()
```

Comment:
I think I meant to delete it but missed it because of the legit comment above it. Good catch.